### PR TITLE
[97] Add schemes and update README.md

### PR DESCRIPTION
**Pull request by nmint8m**
https://github.com/nimbl3/ios-templates/issues/97

## What happened

- [x] Add schemes to template 
- [x] Configure build configurations for schemes
- [x] Update README.md for the post-actions to configure schemes after creating a project with this template
   - Add blueprint identifiers
   - Remove the folder `Removable Resources` 
- [x] Update README.md for the post-actions of build configurations
  - Remove duplicated settings
  - Add note
 
## Insight

- Currently, I leave the blueprint identifier with empty values. I provided post-action in the `README.md` file
- Some references:
   - https://michele.io/the-project-file-part-2/
   - https://www.rubydoc.info/gems/xcodeproj/Xcodeproj/XCScheme/BuildableReference
   - https://developer.apple.com/library/archive/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW106
 
## Proof Of Work

scheme_result

### CODE REVIEW

File: Nimble Templates/App.xctemplate/TemplateInfo.plist [hunk 1]
```diff
@@ -144,8 +156,8 @@
 			<string>___PACKAGENAME___</string>
 			<key>SharedSettings</key>
 			<dict>
-				<key>PRODUCT_NAME</key>
-				<string>$(TARGET_NAME)</string>
+				<!-- <key>PRODUCT_NAME</key>
```
Comment (vnntsu) [inline Nimble Templates/App.xctemplate/TemplateInfo.plist:159]: Please remove it

File: README.md [hunk 2]
```diff
@@ -20,4 +20,68 @@ $ bash install.sh
...
+- `Dev Staging`
+- `Staging`
+- `Dev UAT`
+- `UAT`
+- `Dev Production`
+- `Production`
+
+<img src="./images/readme/configuration_result.png" width="500">
+
+One more thing there are some duplicated build settings which is store in the file `project.pbxproj`. Some of them are automaticcally initialized and some are customized. For an instance, the build settings `PRODUCT_BUNDLE_IDENTIFIER` for `Dev Staging` are duplicated.
```
Comment (vnntsu) [inline README.md:45]: typo `which is`, `automaticcally`, `are duplicated`

File: README.md [hunk 3]
```diff
@@ -20,4 +20,68 @@ $ bash install.sh
...
+
+| Default | Custom |
+|---|---|
+| <img src="./images/readme/configuration_default-attributes.png" width="400"> | <img src="./images/readme/configuration_custom-attributes.png" width="400"> |
+
+What we should do is to remove all duplicated settings for all build configuration. This is the list of settings that you should remove fron the default initializing:
```
Comment (vnntsu) [inline README.md:51]: typo `configuration`, `fron`

File: README.md [hunk 4]
```diff
@@ -20,4 +20,68 @@ $ bash install.sh
...
+
+- `PRODUCT_BUNDLE_IDENTIFIER`
+- `PRODUCT_NAME`
+
+> Note: 
+>
+> - Check the `Debug`/`Release` build configurations are completely remove out of the file `project.pbxproj
```
Comment (vnntsu) [inline README.md:58]: missing ` at the end

**Comment by nmint8m** [comment]
@vnntsu Please take a look 👁️ 👁️  Thank you 💯

**Review by phongvhd93 — APPROVED** [review]

**Review by vnntsu — APPROVED** [review]
LGTM

**Pull request merged**